app/Models/Training/WaitingList.php

```diff
@@ -61,6 +105,10 @@ public function staff()
 
     /**
      * Many WaitingLists can have many Accounts (pivot).
+     *
+     * @fixme remove when no longer used in filament stuff, use `waitingListAccounts` instead.
```

Comment:
Does this still apply?

Reply:
yes i think so, i believe this is used somewhere within adding / removing someone in filament, i'll have another look